WalletWasabi.Fluent/ViewModels/AddWallet/RecoverWalletViewModel.cs

```diff
@@ -106,7 +110,7 @@ private void ValidateMnemonics(IValidationErrors errors)
 			return;
 		}
 
-		errors.Add(ErrorSeverity.Error, "Recovery Words are not valid.");
+		errors.Add(ErrorSeverity.Error, "Invalid Recovery Words. Confirm they're spelled right, sequenced correctly, and aligned with the total count you have.");
```

Comment:
We don't need to mention spelling because the user cannot type any word out of the [2048 english bip 39 word list](https://github.com/bitcoin/bips/blob/master/bip-0039/english.txt).

```suggestion
		errors.Add(ErrorSeverity.Error, "Invalid checksum. Type all your recovery words in the correct order.");
```

Reply:
Strong nack for `checksum` for obvious reasons.